deploy/example/modifyvolume/pod-modifyvolume.yaml

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
pls use existing sc/pvc/pod config yaml files, this yaml file is not necessary

Reply:
But the existing sc has no yaml files to use PremiumV2_LRS which supports DiskIOPSReadWrite and DiskMBpsReadWrite parameters as an example.

Reply:
then only create a new storage class

Reply:
done